src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/common/transpose.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "single_layer_tests/classes/transpose.hpp"
+#include "shared_test_classes/single_layer/transpose.hpp"
+#include "test_utils/cpu_test_utils.hpp"
+
+using namespace InferenceEngine;
+using namespace CPUTestUtils;
+using namespace ngraph::helpers;
+using namespace ov::test;
+
+namespace CPULayerTestsDefinitions {
+namespace Transpose {
+std::map<std::string, std::string> additional_config;
+
+const auto cpuParams_nhwc = CPUSpecificParams {{nhwc}, {}, {}, {}};
+const auto cpuParams_nchw = CPUSpecificParams {{nchw}, {}, {}, {}};
+
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+        Precision::I8,
+        Precision::FP32
+};
+
+const std::vector<std::vector<size_t>> inputOrderPerChannels4D = {
+        std::vector<size_t>{0, 1, 2, 3},
+        std::vector<size_t>{0, 2, 1, 3},
+        std::vector<size_t>{1, 0, 2, 3},
+        std::vector<size_t>{},
+};
+
+const std::vector<CPUSpecificParams> CPUParams4D = {
+        cpuParams_nchw,
```

Comment:
Why do we not test nhwc?
If there is a good reason, please leave a comment in the code.

Reply:
nhwc was not presented in the original file.
nhwc layout is tested using the vector `cpuParams_nhwc`